### PR TITLE
Replace pullup tracker with bozzcoin tracker.

### PR DESCRIPTION
**Pull request by fonglh**
New year, new tracking for unhealthy food!

### CODE REVIEW

File: scripts/bozzcoin.coffee [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    res.send "*#{bozzcoinBalance}* :bozzcoin:"
+    res.send bozzcoinSummaries()
+
+  robot.respond new RegExp("did (-?\\d+) (pullups|pushups|situps)", "i"), (res) ->
+    username = res.message.user.name
+    repsDone = Number.parseInt(res.match[1])
+    exerciseType = res.match[2]
+    switch exerciseType
+      when "pushups", "situps"
+        if repsDone > 60 or repsDone < -60
```
Comment (abeltay) [inline scripts/bozzcoin.coffee]: Why is `repsDone < -60` needed for the response?

Comment (jeremyyap) [reply scripts/bozzcoin.coffee]: I think negative values are for undoing in case of error

Comment (abeltay) [reply scripts/bozzcoin.coffee]: The message seems inaccurate if it's used for undoing.

Comment (naomilwx) [reply scripts/bozzcoin.coffee]: shouldn't the undoing be up to the person's current number of reps?

Comment (fonglh) [reply scripts/bozzcoin.coffee]: @naomilwx good idea

**Review by abeltay — APPROVED** [review]

**Pull request merged**